packages/ihe-gateway-sdk/src/common/shared.ts

```diff
@@ -0,0 +1,12 @@
+const OID_REGEX = /^(urn:oid:)?([0-9]+(\.[0-9]+)*)$/;
+
+export function normalizeOid(oid: string | undefined): string {
+  if (!oid) throw new Error("OID must be present");
+
+  const match = OID_REGEX.exec(oid);
+  if (match && match[2]) {
+    return match[2];
+  }
+
+  throw new Error("Check the OID to make sure it conforms to the proper format: `1.22.333.444`");
```

Comment:
nit: Usually error messages indicate what happened, not what to do.

Could rewrite like `Invalid OID - it should be a series of integers separated by periods`.

This indicates what happens and what's expected. Although adding something like `1.22.333.444` is visually good, it can lead to the reader thinking one needs that amount of "series" and digits.